src/server/schema/resolvers/mutations/createLife.ts

```diff
@@ -0,0 +1,36 @@
+import { ObjectId } from 'mongodb';
+import { GraphQLMutationResolvers, GraphQLLife } from '../definitions';
+import { getDatabaseContext, Life } from '../../../database';
+
+const mutation: GraphQLMutationResolvers['createlife'] = async (
+    root,
+    { firstName, lastName, fullName, title, description, birthday, hobbies }
+) => {
+    const { collections } = await getDatabaseContext();
+    const lifeObject: GraphQLLife = {
+        firstName: firstName,
+        lastName: lastName,
+        fullName: fullName,
+        title: title,
+        birthday: birthday,
+        description: description,
+        hobbies: hobbies,
```

Comment:
You can actually just spread the arguments here or use it directly since they have the same field names

Reply:
Noted! I have changed this to use the arguments directly instead for readability.